static/app/views/performance/newTraceDetails/traceRenderers/traceView.tsx

```diff
@@ -1,8 +1,78 @@
 import {mat3} from 'gl-matrix';
 
+import clamp from 'sentry/utils/number/clamp';
+
 // Computes the transformation matrix that is used to render scaled
 // elements to the DOM and draw the view.
 export class TraceView {
```

Comment:
This was moved from the manager since the view is a separate concept that is not related to the virtualized list.